src/main/java/build/buildfarm/Cat.java

```diff
@@ -607,7 +621,150 @@ static int deadlineSecondsForType(String type) {
     return 10;
   }
 
+  /**
+   * Transform worker string from "ip-10-135-31-210.ec2:8981" to "10-135-31-210".
+   *
+   * @param worker
+   * @return
+   */
+  private static String workerStringTransformation(String worker) {
+    String result = worker.split("\\.")[0].substring("ip-".length()).replaceAll("-", ".") + ":8981";
```

Comment:
Like, I know what this is from? but this is not a job for cat. Please come up with your own sed to do this outside, and pass it in as a parameter.

Reply:
I click `resolve` because i move the new thing to a separate class. Yeah, I definitely will make this string format change out in the future, but for now I didn't see anything else that would potentially use it. 

Also, not super sure what ` come up with your own sed to do this outside` mean here. This is a IP string format change to make grpc call work, the old worker ip string comes from redis directly, not passed in by cmd args.